cmip6_downscaling/workflows/paths.py

```diff
@@ -450,3 +448,103 @@ def make_maca_output_path(gcm_identifier: str, label: str, **kwargs):
         MACA output path
     """
     return f"maca_output/{gcm_identifier}_{label}.zarr"
+
+
+def make_pyramid_path(gcm_identifier: str, **kwargs) -> str:
+    """Build the path for the pyramid
+
+
+    Parameters
+    ----------
+    gcm_identifier : str
+        Output from build_gcm_identifier. String to identify the gcm dataset used
+
+    Returns
+    -------
+    pyramid_path: str
+        Path to gcm_predict ds file location
+    """
+    return f"/pyramid/{gcm_identifier}.zarr"
+
+
+# ---addl bcsd paths
+
+
+def make_return_obs_path(obs_identifier: str, **kwargs) -> str:
+    """Build the path for return obs task
+
+    Parameters
+    ----------
+    obs_identifier : str
+        Output from build_obs_identifier. String to identify the observation dataset used
+
+    Returns
+    -------
+    obs_ds path: str
+        Path to bcsd obs_ds file location
+    """
+    return f"obs_ds/{obs_identifier}.zarr"
+
+
+def make_spatial_anomalies_path(obs_identifier: str = None, **kwargs) -> str:
```

Comment:
```suggestion
def make_spatial_anomalies_path(obs_identifier: str, **kwargs) -> str:
```

Not sure there is any value to make this an optional parameter.